bokehjs/src/lib/models/scales/scale.ts

```diff
@@ -50,4 +50,50 @@ export abstract class Scale extends Transform {
     else
       return [this.invert(sx0), this.invert(sx1)]
   }
+
+  // These are here on Scale because TypeScript does not support mixins. These
```

Comment:
It does support mixins, just differently from what one would do in Python. `Signalable` is a great example of such.